### ISSUE TITLE
BT.2020 Target gamut

### ISSUE THREAD

**Issue by nicko88** [ledoge/novideo_srgb#22]
Could BT.2020 target gamut be added for home theater use?

**Comment by ledoge** [comment]
Sure, but can you explain your specific use case where this would make sense? I didn't add it because then you'd have most colors clipping in SDR, and the only way I could see this option being useful is if you've got a media player that's outputting HDR content transformed to a "normal" gamma curve as SDR.

**Comment by nicko88** [comment]
> the only way I could see this option being useful is if you've got a media player that's outputting HDR content transformed to a "normal" gamma curve as SDR.

This is exactly how many people run their HDR content to their projector.  It's called SDR.2020 and it's using tone-mapping on the PC, usually in the video renderer called madVR (but there are others like JRiverVR and MPV that can do it too).

You tone-map HDR into SDR 2.2 gamma with BT.2020 color space and put your projector into its BT.2020 HDR color space with SDR gamma.

Even commercial devices like the Panasonic UB420/820 UHD players have an output mode like this as you can see here:

https://www.avsforum.com/attachments/img_7592-jpg.2517434/

Not that this has any bearing on this software, but just an example that it's a somewhat prevalent mode.  Video processors like Lumagen and madVR Envy also have output modes for this.

Projectors rarely go above 100 nit, and handle HDR poorly, so external tone-mapping to "SDR.2020" is a very handy mode.

**Comment by ledoge** [comment]
Very interesting, thanks! I'll add it then.

**Comment by ledoge** [comment]
Just pushed a new release with it added :)

**Comment by nicko88** [comment]
I see, that was fast, thanks!

I figured it was a fairly easy add and I know I will use this for me or some of my HT friends, most likely the guy with the $100K Christie Griffyn with the 100% BT.2020 RGB lasers.

**Comment by James-F2** [comment]
I confirm everything works correctly.
When madvr and novideo_srgb set to 2020 the colors look accurate in 709 and P3 content (HD and UHD blurays).
BT.2020 has benefit over P3 because some displays/projectors go beyond P3 and UHD blurays often have colors outside P3.
👍

**Comment by nicko88** [comment]
Yeah, with the advent of QD-OLED we are really starting to venture into mainstream displays > 100% P3 so it's nice to have BT.2020 support, at least for video.
